lib/posix-process/process.h

```diff
@@ -74,6 +74,9 @@ for (int _j = 1, _i = 0; _i != ARRAY_SIZE(pid_process);			\
 	uk_list_for_each_entry_safe(_pthread, _pthreadn,		\
 				    &(_proc)->threads, thread_list_entry)
 
+#define uk_gettid()	ukthread2tid(uk_thread_current())
```

Comment:
Yes, the reason I used a `#define` here was because this was meant to be simple aliasing. As you said, the assertion should really be at `ukthread2tid`. I'll add a commit to do that in this PR.